app/code/Magento/Catalog/view/adminhtml/web/catalog/product/attribute/unique-validate.js

```diff
@@ -25,21 +31,27 @@ define([
                             .closest('table')
                             .find('.' + _config.uniqueClass + ':visible'),
                         valuesHash = {},
-                        isValid = true;
+                        isValid = true,
+                        duplicates = [];
 
                     inputs.each(function (el) {
                         var inputValue = inputs[el].value;
 
                         if (typeof valuesHash[inputValue] !== 'undefined') {
                             isValid = false;
+                            duplicates.push(inputValue);
                         }
                         valuesHash[inputValue] = el;
                     });
 
+                    if (!isValid) {
+                        msg = _config.message.replace('%1', duplicates.join(', '));
+                    }
+
                     return isValid;
                 },
 
-                _config.message
+                messager
```

Comment:
@PieterCappelle Why you can't use `msg` variable? Why you declare `messager` function?